catalogs/data/bedrock/bedrock-servicedeployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: deployments.plural.sh/v1alpha1
```

Comment:
I'm not sure what this is doing, but you should be deploying the `ai-proxy` helm chart that's already defined in the console repo.